fedbiomed/common/secagg/_jls.py

```diff
@@ -117,7 +120,14 @@ def divide(xs: List[int], k: int) -> List[int]:
     Returns:
         List of divided integers
     """
```

Comment:
@mvesin there is also a `multiply` function which converts to `uint32`. 
Do you think we should also change it to `uint64`?

Reply:
Good question, thank you @sharkovsky !

I wondered whether to robustify also this function, but did not do it because we have "some" guarantees on the way the values are built in `SecaggCrypter.encrypt()`

But at second read I fully agree with you it would be more coherent with the other proposed changes to include it !

Rather than changing the type to `uint64` I would propose to add a check to the received value (that it is in the [0, 2**32-1] range) : in this case we don't need to handle bigger values, we just need to check the received value correspond to what is expected.